services/management/checks.go

```diff
@@ -131,3 +135,28 @@ func (s *ChecksAPIService) ChangeSecurityChecks(req *managementpb.ChangeSecurity
 
 	return &managementpb.ChangeSecurityChecksResponse{}, nil
 }
+
+// parses the check script and returns the docstring for the `check_context` function.
+func (s *ChecksAPIService) getCheckDescription(check check.Check) string {
+	// TODO There is similar code in check service; move this to a common package if possible.
+	// https://jira.percona.com/browse/SAAS-429
+	var thread starlark.Thread
+	globals, err := starlark.ExecFile(&thread, "", check.Script, nil)
```

Comment:
We should pass our predefined functions there (`GetFuncsForVersion(1)`)

Reply:
blocked by https://github.com/percona-platform/platform/pull/160

Reply:
saas repo updated